evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -147,10 +135,6 @@ <h5 class="modal-title" id="previewModalLabel">{% trans 'Preview' %}</h5>
         };
     </script>
 
-    {% blocktrans asvar title with evaluation_name=evaluation.full_name|safe %}Request participant changes for {{ evaluation_name }}{% endblocktrans %}
-    {% trans 'Please tell us what changes to the participants list we should make.' as teaser %}
-    {% include 'contact_modal.html' with modal_id='changeParticipantRequestModal' user=request.user title=title teaser=teaser %}
-
     {% trans 'Request account creation' as title %}
```

Comment:
in case managers are asked to add the newly created accounts to the evaluation, this gives them the required info where they belong:
```suggestion
    {% blocktrans asvar title with evaluation_name=evaluation.full_name|safe %}Request account creation for {{ evaluation_name }}{% endblocktrans %}
```